src/handlers/FlingGestureHandler.ts

```diff
@@ -46,14 +61,13 @@ export const flingHandlerName = 'FlingGestureHandler';
 
 export type FlingGestureHandler = typeof FlingGestureHandler;
 // eslint-disable-next-line @typescript-eslint/no-redeclare -- backward compatibility; see description on the top of gestureHandlerCommon.ts file
-export const FlingGestureHandler = createHandler<
-  FlingGestureHandlerProps,
-  FlingGestureHandlerEventPayload
->({
-  name: flingHandlerName,
-  allowedProps: [
-    ...baseGestureHandlerProps,
-    ...flingGestureHandlerProps,
-  ] as const,
-  config: {},
-});
+export const FlingGestureHandler = !isMacOS()
+  ? createHandler<FlingGestureHandlerProps, FlingGestureHandlerEventPayload>({
+      name: flingHandlerName,
+      allowedProps: [
+        ...baseGestureHandlerProps,
+        ...flingGestureHandlerProps,
+      ] as const,
+      config: {},
+    })
+  : FlingFallback;
```

Comment:
Is it necessary? I mean, does the app crash if you use `Fling`? If not, we could simply add Xcode warn,[ like here](https://github.com/software-mansion/react-native-gesture-handler/blob/8c7b066bdb7ff0d1fb87f10fe9200ed29085f3f8/ios/Handlers/RNHoverHandler.m#L88).

Cc @j-piasecki, what do you think?

Reply:
Yes, both me and @j-piasecki agreed on that.